signer/src/storage/in_memory.rs

```diff
@@ -314,7 +314,7 @@ impl super::DbRead for SharedStore {
                     .deposit_request_to_signers
                     .get(&(deposit_request.txid, deposit_request.output_index))
                     .map(|signers| {
-                        signers.iter().filter(|signer| signer.is_accepted).count() >= threshold
+                        signers.iter().filter(|signer| signer.can_accept).count() >= threshold
```

Comment:
Should this now be `signers.iter().filter(|signer| signer.can_accept && signer.can_sign).count() >= threshold` to match the Postgres implementation behavior?

Reply:
If that's the case, we should update the docstring (and maybe the name) of `get_pending_accepted_deposit_request` to specify that it returns requests that can be both accepted and signed to avoid possible confusion!

https://github.com/stacks-network/sbtc/blob/c16e287bdfc39cbe21c17083eff2939748d008cc/signer/src/storage/mod.rs#L62-L64

Reply:
Fixed them both up in https://github.com/stacks-network/sbtc/pull/838/commits/c00ac5404f4c3a32cb80f04a7a4116c897be4178